### PR TITLE
add Nerd Tree Patched Font File Type Icons

### PR DESCRIPTION
**Pull request by rocknrollMarc**
Very nice!!! for example you get red file names for ruby with a red diamond in Nerdtree and in lightline...sexy.
I added a submudule as you can choose from some very cool pathced fonts.
You can patch every personal font with the included ./ fontpacher script.

take a look at these repos for info:

* https://github.com/ryanoasis/nerd-filetype-glyphs-fonts-patcher.git
* https://github.com/ryanoasis/vim-webdevicons.git

Greetings and Rocknroll from Hamburg Germany

### CODE REVIEW

**Comment by rocknrollMarc** [comment]
overall-screenshot

**Comment by rocknrollMarc** [comment]
screen shot 2015-04-28 at 10 27 53

File: .vim [hunk 1]
```diff
@@ -0,0 +1 @@
+dotvim
```
Comment (vitaly) [inline .vim]: what is this? :)

File: .vimrc.after [hunk 2]
```diff
@@ -0,0 +1,28 @@
+colorscheme solarized
```
Comment (vitaly) [inline .vimrc.after]: file `.vimrc.after` is intended for local overriders, not for the generic `dotvim` repository

File: before.vim [hunk 3]
```diff
@@ -13,3 +13,6 @@ if has('gui_running')
     source ~/.gvimrc.before
   endif
 endif
+
+
+colorscheme solarized
```
Comment (vitaly) [inline before.vim]: local setting. shouldn't be here.

**Comment by vitaly** [comment]
it seems like the change is based on your local changes to dotvim, with too many unrelated changes. can you please extract the relevant part into a separate branch based off of current master?

**Comment by rocknrollMarc** [comment]
Yap still strugeling a bitz with that ;) sorry Ill fix as soon as I have 2 minutes

**Comment by ryanoasis** [comment]
And if you need anything from me let me know :smile:

**Comment by rocknrollMarc** [comment]
Well as you are asing how would I go about this :) I have this often and dont really have the time to do all repos I commit to. Can I do this on giuthub interface or do I have to squash my commits per hand ?

**Comment by ryanoasis** [comment]
@rocknrollMarc Not sure if your comment was meant for me? If it was I think what is being discussed here can more easily be done command line or at least thats how I usually do it unless its very trivial

**Pull request closed without merging**